docs/modules/operation/pages/provisioning/detectors.adoc

```diff
@@ -1,20 +1,8 @@
 [[ga-detectors]]
 = Detectors
 
-Detectors can be used within the provisioning process to detect available services on nodes automatically.
-
-[[ga-provisioning-meta-data]]
-== Metadata DSL
-Service detectors can use <<meta-data.adoc#ga-meta-data-dsl, MetaData DSL>> (domain specific language) to leverage dynamic configuration in parameter values to interpolate metadata into the parameter.
-
-The syntax lets you use patterns in an expression, whereby the metadata is replaced with a corresponding value during the collection process.
-
-During evaluation of an expression, the following scopes are available:
-
-* Node metadata
-* Interface metadata
-
+Use detectors within the provisioning process to detect available services on nodes automatically.
 
 == Supported detectors
 
-The supported detectors can be found in xref:reference:provisioning/detectors[provisioning detectors reference section].
\ No newline at end of file
+For information on supported detectors and how to configure them, see xref:reference:configuration/provisioning/introduction.adoc[provisioning detectors reference section].
```

Comment:
```suggestion
For information on supported detectors and how to configure them, see xref:reference:configuration/provisioning/introduction.adoc#ref-detectors[provisioning detectors reference section].
```

Reply:
I would suggest keeping the MetaData text here in the operation section too.
```suggestion
For information on supported detectors and how to configure them, see xref:reference:configuration/provisioning/introduction.adoc[provisioning detectors reference section].54

[[ga-detector-provisioning-meta-data]]
== Metadata DSL
Service detectors can use <<:meta-data.adoc#ga-meta-data-dsl, MetaData DSL>> (domain specific language) to leverage dynamic configuration in parameter values to interpolate metadata into the parameter.

The syntax lets you use patterns in an expression, whereby the metadata is replaced with a corresponding value during the collection process.

During evaluation of an expression, the following scopes are available:

* Node metadata
* Interface metadata
```